src/public/oauth-authorize.html

```diff
@@ -0,0 +1,108 @@
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width">
+
+		<title>Authorize application - Pokémon Showdown!</title>
+
+		<link rel="stylesheet" href="//pokemonshowdown.com/style/global.css">
+	</head>
+	<body>
+		<div class="body">
+		<header>
+			<div class="nav-wrapper"><ul class="nav">
+				<li><a class="button nav-first" href="/"><img src="/images/pokemonshowdownbeta.png" srcset="/images/pokemonshowdownbeta.png 1x, /images/pokemonshowdownbeta@2x.png 2x" alt="Pokémon Showdown" width="146" height="44"> Home</a></li>
+				<li><a class="button" href="/dex/">Pokédex</a></li>
+				<li><a class="button" href="//replay.pokemonshowdown.com/">Replays</a></li>
+				<li><a class="button" href="/ladder/">Ladder</a></li>
+				<li><a class="button nav-last" href="/forums/">Forum</a></li>
+				<li><a class="button greenbutton nav-first nav-last" href="//play.pokemonshowdown.com/">Play</a></li>
+			</ul></div>
+		</header>
+
+		<div class="main"><section class="section">
+			<h2>Authorize {{client}}</h2><hr />
+			<div class="infobox">
+				{{client}} by {{client_name}} wants to access your Pokemon Showdown account.<br />
+				This is public data only, and will allow you to log in on their site using your Pokemon Showdown account for two weeks.<br /><br /><hr />
```

Comment:
Probably it would be good to be more specific about what "This is public data only" means.

Reply:
DId.